### DISCUSSION TITLE
Is Evrete suitable for use in authorization management?

### DISCUSSION THREAD

**Discussion by eastack** [evrete/evrete#109] [General]
I’ve been researching how to implement authorization management using Datalog recently, but I haven't found any particularly good libraries for Java. Then I came across Evrete, which seems promising, but I’m still not very familiar with it and am unsure whether it’s suitable for use in authorization management.

**Comment by andbi** [comment]
Hi,

Personally, I don't foresee any conceptual problems implementing Datalog-like logic in Evrete as both are forward-chaining environments. You'll need to do some parsing, possibly employing ANTLR4, that's for sure, but otherwise, it's doable.
